discretize/utils/interputils.py

```diff
@@ -66,99 +66,3 @@ def interpmat(locs, x, y=None, z=None):
     Q = sp.csr_matrix((vals, (I, J)),
                       shape=(npts, np.prod(shape)))
     return Q
-
-if not _interpCython:
```

Comment:
Until we get the pip install stable with cython, I think we should hang on to this code...